### PR TITLE
Allow sandbox to take local path in addition to url 

### PR DESCRIPTION
**Pull request by cgewecke**
This makes it easier to use a wider variety of target projects in the tests.

### CODE REVIEW

File: lib/utils/unbox.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function checkDestination(destination) {
     var contents = fs.readdirSync(destination);
     if (contents.length) {
       var err = "Something already exists at the destination. " +
-                "Please unbox in an empty folder. " +
+                "Please run `truffle init` or `truffle unbox` in an empty folder. " +
```
Comment (cgewecke) [inline lib/utils/unbox.js:19]: Also fixed this message, which has been the source of understandable bewilderment. People see this when they `truffle init`. Talking about boxes here is a little confusing for them.

**Comment by gnidan** [comment]
@cgewecke is that scenario test failure significant here?

**Comment by cgewecke** [comment]
@gnidan No, only there.

**Comment by cgewecke** [comment]
@gnidan Convinced me not to do this.

**Pull request closed without merging**